examples/us_street_single_address_example.rb

```diff
@@ -40,11 +40,14 @@ def run
     lookup.state = 'CA'
     lookup.zipcode = '21229'
     lookup.candidates = 3
+    lookup.county_source = SmartyStreets::USStreet::CountySource::GEOGRAPHIC
```

Comment:
Do we want this to be a default for the example?

Reply:
I would presume so. The reason we use match=invalid as the default example for that parameter is due to enhanced being locked behind a paywall. We could comment it out if we'd like to

Reply:
It's probably ok.